.storybook/pages/CadModal/ModalFinalStep.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Button, Modal, Text } from '../../../src';
+
+import '../../../src/components/Utilities/Alignment.css';
+import '../../../src/components/Utilities/Display.css';
+
+import CadSuccess from '../../static/cad-success.svg';
+
+export const ModalFinalStep = () => (
+  <Modal
+    onClose={
+      () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+    }
+    open
+  >
+    <Modal.Header>
+      <div
+        className="u-display-flex u-flex-direction-column u-justify-content-center u-align-items-center"
+        style={{ height: '25rem' }}
+      >
+        <img alt="cad-success" src={CadSuccess} />
```

Comment:
How about "a checkmark"? Sorry to be nitty about alt text but I'm kind of expecting that it will be copied over wholesale into the product, so I'd like it to be passable even if it's not ideal. (Ideally this would be written up by design/content specialists in the designs imo.)

Reply:
Changed alt, thanks